p503/isogeny_test.go

```diff
@@ -0,0 +1,98 @@
+package p503
+
+import (
+	. "github.com/cloudflare/p751sidh/internal/isogeny"
+	"testing"
+)
+
+func TestFourIsogenyVersusSage(t *testing.T) {
```

Comment:
I think it would be good to have the Sage code here that you used to generate these values, like for p751.

Reply:
it would be useful to have it but only if it is correct. the one in p751 surely isn't